### DISCUSSION TITLE
Replace Butterfly with SkyWalking

### DISCUSSION THREAD

**Discussion by aioloswong** [ThreeMammals/Ocelot#2104] [Ideas]
## Expected Behavior / New Feature
As butterfly is no longer being maintained，please replace it with apache skywalking. 
https://github.com/SkyAPM/SkyAPM-dotnet
https://github.com/apache/skywalking

**Comment by raman-m** [comment]
Will you personally contribute to release a Tracing provider NuGet package for Ocelot?

-----
> https://github.com/SkyAPM/SkyAPM-dotnet

[Releases](https://github.com/SkyAPM/SkyAPM-dotnet/releases) &rarr; [v2.1.0-release](https://github.com/SkyAPM/SkyAPM-dotnet/releases/tag/v2.1.0) is the latest one on Sep 27, 2022
So, this repo is not supported also :warning: 

----
> https://github.com/apache/skywalking

But this is Java product! Some drivers?
I've checked SkyWalking docs and there is no .NET agent at all! See: [Agents](https://skywalking.apache.org/docs/#Agents)

**Comment by raman-m** [comment]
@RaynaldM @ggnaegi  This is Tracer and monitoring product

https://github.com/apache/skywalking
Apache SkyWalking is amazing product! It is coded in Java! So, an agent lib is required, but there is no .NET [agent](https://skywalking.apache.org/docs/#Agents) :cry:
